packages/lms/src/lib/components/navigation/Entity.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+    import { drawerStore } from '@skeletonlabs/skeleton';
```

Comment:
some funky stuff going on with indentation in this file. could be related and fixed with #165

Reply:
I did some indentation stuff and played around with it - get back it's equal after normalization. So it's ignoring compiler output. 
Might be something else i need to look at though - or as you say might be related to 165